pandas/core/missing.py

```diff
@@ -481,6 +481,10 @@ def pad_1d(values, limit=None, mask=None, dtype=None):
         _method = algos.pad_inplace_float64
     elif values.dtype == np.object_:
         _method = algos.pad_inplace_object
+    elif is_timedelta64_dtype(values):
+        # NaTs are treated identically to datetime64, so we can dispatch
```

Comment:
is something failing? i agree this is probably the soln. but likely these are converted prior to calling this (to i8)

Reply:
Presumably; I haven’t looked at the series code to see how it handles this.  Enough core.missing mysteries have popped up today I’ll be giving them a close look after the RC

Reply:
sure